lib/events/uploader.go

```diff
@@ -38,6 +38,16 @@ type UploadHandler interface {
 	UploadSummary(ctx context.Context, sessionID session.ID, readCloser io.Reader) (string, error)
 	// DownloadSummary downloads a session summary and writes it to a writer.
 	DownloadSummary(ctx context.Context, sessionID session.ID, writer RandomAccessWriter) error
+	// UploadMetadata uploads session metadata and returns a URL with uploaded
+	// file in case of success.
+	UploadMetadata(ctx context.Context, sessionID session.ID, readCloser io.Reader) (string, error)
+	// DownloadMetadata downloads session metadata and writes it to a writer.
+	DownloadMetadata(ctx context.Context, sessionID session.ID, writer RandomAccessWriter) error
+	// UploadThumbnail uploads a session thumbnail and returns a URL with uploaded
+	// file in case of success.
+	UploadThumbnail(ctx context.Context, sessionID session.ID, readCloser io.Reader) (string, error)
+	// DownloadThumbnail downloads a session thumbnail and writes it to a writer.
+	DownloadThumbnail(ctx context.Context, sessionID session.ID, writer RandomAccessWriter) error
```

Comment:
Could we expand these commands just a little bit? Probably the summary ones too.

```
// UploadMetadata uploads session metadata and returns a URL with the uploaded
// file in case of success. Session metadata is a JSON file containing basic
// information about the session recording.
```

(And then similarly describe what a thumbnail is. Is it an image file?)